.changeset/light-dryers-talk.md

```diff
@@ -0,0 +1,7 @@
+---
+'lit-html': minor
+'@lit/reactive-element': minor
+'lit': patch
```

Comment:
Shouldn't `lit` be minor too or no?

Reply:
I was a bit torn about it.. It's not changing the `lit` package directly, only its deps, but since we consider these deps core to `lit`, I think minor does make more sense.